openapi-plugin/src/main/java/io/micronaut/gradle/openapi/MicronautOpenApiPlugin.java

```diff
@@ -23,7 +23,7 @@
 
 import java.util.List;
 
-public class MicronautOpenAPIPlugin implements Plugin<Project> {
+public class MicronautOpenApiPlugin implements Plugin<Project> {
```

Comment:
:x: This is a breaking change. Please revert this, as it is a public class and nothing prevents another plugin from applying it by class literal. This is in fact the recommended approach when composing plugins.

Reply:
@melix ok, got it! Reverted